plugins/package-managers/cocoapods/build.gradle.kts

```diff
@@ -37,6 +40,9 @@ dependencies {
     implementation(projects.utils.ortUtils)
     implementation(projects.utils.spdxUtils)
 
+    implementation(libs.kotlinx.serialization.core)
+    implementation(libs.kotlinx.serialization.json)
+
     implementation(libs.jackson.annotations)
```

Comment:
The Jackson dependencies should be removed.

Reply:
Not yet. The lockfile parsing also needs to be ported.